### PR TITLE
[DOCS] Fix relative links on github

### PR DESCRIPTION
**Pull request by andyjessen**
This commit fixes relative links returning '404' error when accessed on https://github.com/great-expectations/great_expectations.

### Definition of Done
Please delete options that are not relevant.

- [X] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/docs/contributing/style_guides/code_style)
- [X] I have performed a [self-review](https://docs.greatexpectations.io/docs/contributing/contributing_checklist) of my own code

### CODE REVIEW

File: docs/intro.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ Slack is good for that, too: [join slack](https://greatexpectations.io/slack).
 If you have bugfix or feature request, please [upvote an existing issue](https://github.com/great-expectations/great_expectations/issues) or [open a new issue](https://github.com/great-expectations/great_expectations/issues/new) on GitHub and we’ll see what we can do.
 
 #### Contribute code or documentation
-We welcome contributions to Great Expectations. Please start with our [Contributing guide](/docs/contributing/contributing) and don’t be shy with questions!
+We welcome contributions to Great Expectations. Please start with our [Contributing guide](../contributing.md) and don’t be shy with questions!
```
Comment (austiezr) [inline docs/intro.md:104]: This should be `./contributing/contributing.md`

Comment (austiezr) [inline docs/intro.md]: This should be `./contributing/contributing.md`

File: docs/tutorials/getting_started/customize_your_deployment.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 title: Optional - Customize your deployment
 ---
 
-At this point, you have your first, working local deployment of Great Expectations. You’ve also been introduced to the foundational concepts in the library: [Data Contexts](/docs/reference/data_context), [Datasources](/docs/reference/datasources), [Expectations](/docs/reference/expectations/expectations), [Profilers](/docs/reference/profilers), [Data Docs](/docs/reference/data_docs), [Validation](/docs/reference/validation), and [Checkpoints](/docs/reference/checkpoints_and_actions).
+At this point, you have your first, working local deployment of Great Expectations. You’ve also been introduced to the foundational concepts in the library: [Data Contexts](../../reference/data_context.md), [Datasources](../../reference/datasources.md), [Expectations](../../reference/expectations/expectations.md), [Profilers](../../reference/profilers.md), [Data Docs](../../reference/data_docs.md), [Validation](../../docs/reference/validation.md), and [Checkpoints](../../reference/checkpoints_and_actions.md).
```
Comment (austiezr) [inline docs/tutorials/getting_started/customize_your_deployment.md]: Validation link should be `../../reference/validation.md`

File: docs/tutorials/getting_started/customize_your_deployment.md [hunk 3]
```diff
@@ -15,88 +15,88 @@ This last section of this tutorial is designed to present you with clear options
...
-* [How to configure an Expectation store in GCS](/docs/guides/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_in_gcs)
-* [How to configure an Expectation store in Azure Blob Storage](/docs/guides/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_in_azure_blob_storage)
-* [How to configure an Expectation store to PostgreSQL](/docs/guides/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_to_postgresql)
-* [How to configure an Expectation store on a filesystem](/docs/guides/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_to_postgresql)
+* [How to configure an Expectation store in Amazon S3](../../guides/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_in_amazon_s3.md)
+* [How to configure an Expectation store in GCS](../../guides/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_in_gcs.md)
+* [How to configure an Expectation store in Azure Blob Storage](../..//guides/setup/configuring_metadata_stores/how_to_configure_an_expectation_store_in_azure_blob_storage.md)
```
Comment (austiezr) [inline docs/tutorials/getting_started/customize_your_deployment.md]: Extra `/`

File: docs/contributing/contributing_checklist.md [hunk 4]
```diff
@@ -48,9 +48,9 @@ Once your code is ready, please go through the following checklist before submit
 If your PR contains **only** changes to `contrib` directory (community contributed Expectations), please skip this step - the tests that you included in the Expectation itself are sufficient and you do not need to run pytest.
 :::
 
-* See [Writing unit and integration tests](/docs/contributing/contributing_test) for details on how to write unit tests in Great Expectations.
+* See [Writing unit and integration tests](../contributing/contributing_test.md) for details on how to write unit tests in Great Expectations.
 
-* Please make certain to run `pytest` to verify that all tests pass locally. See [Running tests](/docs/contributing/contributing_test) for details.
+* Please make certain to run `pytest` to verify that all tests pass locally. See [Running tests](../contributing/contributing_test.md) for details.
```
Comment (austiezr) [inline docs/contributing/contributing_checklist.md]: `./contributing_test.md`

File: docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md [hunk 5]
```diff
@@ -324,8 +324,8 @@ We will now take the local GE configuration from [Part 1](#part-1-local-configur
 
 There are a number of ways that Great Expectations can be run in Cloud Composer or Airflow.
 
-1. [Running a Checkpoint in Airflow using a `bash operator`](/docs/deployment_patterns/how_to_use_great_expectations_with_airflow#option-1-running-a-checkpoint-with-a-bashoperator)
-2. [Running a Checkpoint in Airflow using a `python operator`](/docs/deployment_patterns/how_to_use_great_expectations_with_airflow#option-2-running-the-checkpoint-script-output-with-a-pythonoperator)
+1. [Running a Checkpoint in Airflow using a `bash operator`](../deployment_patterns/how_to_use_great_expectations_with_airflow#option-1-running-a-checkpoint-with-a-bashoperator.md)
+2. [Running a Checkpoint in Airflow using a `python operator`](../deployment_patterns/how_to_use_great_expectations_with_airflow#option-2-running-the-checkpoint-script-output-with-a-pythonoperator.md)
```
Comment (austiezr) [inline docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md]: `./how_to_use_great_expectations_with_airflow#option-1-running-a-checkpoint-with-a-bashoperator.md`

`./how_to_use_great_expectations_with_airflow#option-2-running-the-checkpoint-script-output-with-a-pythonoperator.md`

File: docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md [hunk 6]
```diff
@@ -451,14 +451,14 @@ You've successfully migrated your Great Expectations configuration to Cloud Comp
 
 There are many ways to iterate and improve this initial version, which used a `bash operator` for simplicity. For information on more sophisticated ways of triggering Checkpoints, building our DAGs, and dividing our Data Assets into Batches using DataConnectors, please refer to the following documentation: 
 
-- [How to run a Checkpoint in Airflow using a `python operator`](/docs/deployment_patterns/how_to_use_great_expectations_with_airflow#option-2-running-the-checkpoint-script-output-with-a-pythonoperator).
+- [How to run a Checkpoint in Airflow using a `python operator`](../deployment_patterns/how_to_use_great_expectations_with_airflow#option-2-running-the-checkpoint-script-output-with-a-pythonoperator.md).
```
Comment (austiezr) [inline docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md]: `./how_to_use_great_expectations_with_airflow#option-2-running-the-checkpoint-script-output-with-a-pythonoperator.md`

File: docs/guides/connecting_to_your_data/components/next_steps.md [hunk 7]
```diff
@@ -1,5 +1,5 @@
 Now that you've connected to your data, you'll want to work on these **core skills**:
 
-- [How to create a Batch of data from an in-memory Spark or Pandas dataframe](/docs/guides/connecting_to_your_data/how_to_create_a_batch_of_data_from_an_in_memory_spark_or_pandas_dataframe)
-- [How to create and edit Expectations with instant feedback from a sample Batch of data](/docs/guides/expectations/how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data)
-- [How to create and edit Expectations based on domain knowledge, without inspecting data directly](/docs/guides/expectations/how_to_create_and_edit_expectations_based_on_domain_knowledge_without_inspecting_data_directly)
+- [How to create a Batch of data from an in-memory Spark or Pandas dataframe](../../connecting_to_your_data/how_to_create_a_batch_of_data_from_an_in_memory_spark_or_pandas_dataframe.md)
```
Comment (austiezr) [inline docs/guides/connecting_to_your_data/components/next_steps.md:3]: `../connecting_to_your_data/how_to_create_a_batch_of_data_from_an_in_memory_spark_or_pandas_dataframe.md`

File: docs/guides/miscellaneous/how_to_write_a_how_to_guide.md [hunk 8]
```diff
@@ -128,24 +128,24 @@ Once a draft of your guide is written, you can see it rendered on a `localhost`
...
+1. Ensure that your dev environment is set up according to the guide on [Setting up your dev environment](../../contributing/contributing_setup.md)
 
 2. Copy the How-to guide template file to the appropriate subdirectory of `docs/guides/`, and rename it.
 
 3. Write a title and purpose paragraph.
 
 4. Decide whether you’re writing a code-heavy or process-heavy guide, and adjust your formatting appropriately.
 
-5. Fill out the Prerequisites info box (see [How-to guide template file](/docs/guides/miscellaneous/how_to_template)). The header of the info box says: “This how-to guide assumes you have already:”. Place each prerequisite under its own bullet and phrase it using the style in the template: “done something” (e.g. "Configured a Datasource").
+5. Fill out the Prerequisites info box (see [How-to guide template file](../../guides/miscellaneous/how_to_template.md)). The header of the info box says: “This how-to guide assumes you have already:”. Place each prerequisite under its own bullet and phrase it using the style in the template: “done something” (e.g. "Configured a Datasource").
```
Comment (austiezr) [inline docs/guides/miscellaneous/how_to_write_a_how_to_guide.md]: `./how_to_template.md`

File: docs/guides/miscellaneous/migration_guide.md [hunk 9]
```diff
@@ -201,7 +201,7 @@ The V3-style Datasource has:
 
 </details>
 
-Migrating Datasource configurations that contain connections to the cloud or databases involve additional parameters like credentials that are specific to each configuration. The how-to-guides for Great Expectations contain numerous examples of V3 configurations that can be used for these various situations. Please check out our documentation on [Connecting to your Data](/docs/guides/connecting_to_your_data/index) for examples on V3-style Datasource configurations that will suit your needs.
+Migrating Datasource configurations that contain connections to the cloud or databases involve additional parameters like credentials that are specific to each configuration. The how-to-guides for Great Expectations contain numerous examples of V3 configurations that can be used for these various situations. Please check out our documentation on [Connecting to your Data](../../guides/connecting_to_your_data/index.md) for examples on V3-style Datasource configurations that will suit your needs.
```
Comment (austiezr) [inline docs/guides/miscellaneous/migration_guide.md]: `../connecting_to_your_data/index.md`

File: docs/guides/miscellaneous/migration_guide.md [hunk 10]
```diff
@@ -232,7 +232,7 @@ The V3-style Datasource has:
...
```
Comment (austiezr) [inline docs/guides/miscellaneous/migration_guide.md]: `../connecting_to_your_data/index.md`

File: docs/guides/miscellaneous/migration_guide.md [hunk 11]
```diff
@@ -250,7 +250,7 @@ The V3-style Datasource has:
 
 :::note Note on Datasource in V3
 
-One exception to the datatype-agnostic Datasource in the V3 API is the <code>SimpleSqlalchemyDatasource</code>, which combines functionality of the <code>Datasource</code> and <code>ExecutionEngine</code> to enable [database introspection and partitioning](/docs/guides/connecting_to_your_data/how_to_configure_a_dataconnector_to_introspect_and_partition_tables_in_sql). More examples on using the <code>SimpleSqlalchemyDatasource</code> can be found [here](/docs/guides/connecting_to_your_data/how_to_configure_a_dataconnector_to_introspect_and_partition_tables_in_sql).
+One exception to the datatype-agnostic Datasource in the V3 API is the <code>SimpleSqlalchemyDatasource</code>, which combines functionality of the <code>Datasource</code> and <code>ExecutionEngine</code> to enable [database introspection and partitioning](../../guides/connecting_to_your_data/how_to_configure_a_dataconnector_to_introspect_and_partition_tables_in_sql.md). More examples on using the <code>SimpleSqlalchemyDatasource</code> can be found [here](../../guides/connecting_to_your_data/how_to_configure_a_dataconnector_to_introspect_and_partition_tables_in_sql.md).
```
Comment (austiezr) [inline docs/guides/miscellaneous/migration_guide.md]: `../connecting_to_your_data/<path>`

File: docs/guides/miscellaneous/migration_guide.md [hunk 12]
```diff
@@ -273,12 +273,12 @@ Migrating Datasource configurations that contain connections to databases involv
...
-- [How to connect to a MySQL database](/docs/guides/connecting_to_your_data/database/mysql)
-- [How to connect to a Redshift database](/docs/guides/connecting_to_your_data/database/redshift)
-- [How to connect to a Snowflake database](/docs/guides/connecting_to_your_data/database/snowflake)
-- [How to connect to a SQLite database](/docs/guides/connecting_to_your_data/database/sqlite)
+- [How to connect to a Athena database](../../guides/connecting_to_your_data/database/athena.md)
+- [How to connect to a BigQuery database](../../guides/connecting_to_your_data/database/bigquery.md)
+- [How to connect to a MySQL database](../../guides/connecting_to_your_data/database/mysql.md)
+- [How to connect to a Redshift database](../../guides/connecting_to_your_data/database/redshift.md)
+- [How to connect to a Snowflake database](../../guides/connecting_to_your_data/database/snowflake.md)
+- [How to connect to a SQLite database](../../guides/connecting_to_your_data/database/sqlite.md)
```
Comment (austiezr) [inline docs/guides/miscellaneous/migration_guide.md]: `../connecting_to_your_data/<path>`

File: docs/guides/miscellaneous/migration_guide.md [hunk 13]
```diff
@@ -291,7 +291,7 @@ Before doing the migration, we recommend that you create a backup of your V2 Che
...
+In Great Expectations version 0.13.7, we introduced an improved Checkpoints feature, which allowed Checkpoints to utilize features the V3 API. As a result, Checkpoints are now able to [filter and sort batches from configured datasources](../..//guides/connecting_to_your_data/how_to_get_a_batch_of_data_from_a_configured_datasource.md), [introspect and partition tables as batches](../../guides/connecting_to_your_data/how_to_configure_a_dataconnector_to_introspect_and_partition_tables_in_sql.md), with multi-batch Expectations soon to come.  As part of these design improvements, Validation Operators (originally located in the `great_expectations.yml` file) were combined into Checkpoint configurations.
```
Comment (austiezr) [inline docs/guides/miscellaneous/migration_guide.md]: `../connecting_to_your_data/<path>`

File: docs/guides/miscellaneous/migration_guide.md [hunk 14]
```diff
@@ -349,8 +349,8 @@ Here is the equivalent configuration in V3-style. Notice that the Validation Ope
 
 For additional examples on how to configure V3-style checkpoints, including how to use `test_yaml_config` to build advanced configurations, please refer to our documentation here:
 
-- [How to add validations data or suites to a Checkpoint](/docs/guides/validation/checkpoints/how_to_add_validations_data_or_suites_to_a_checkpoint)
-- [How to configure a new Checkpoint using test_yaml_config](/docs/guides/validation/checkpoints/how_to_configure_a_new_checkpoint_using_test_yaml_config)
+- [How to add validations data or suites to a Checkpoint](../../guides/validation/checkpoints/how_to_add_validations_data_or_suites_to_a_checkpoint.md)
```
Comment (austiezr) [inline docs/guides/miscellaneous/migration_guide.md]: `../validation/<path>`

File: docs/guides/miscellaneous/migration_guide.md [hunk 15]
```diff
@@ -429,8 +429,8 @@ Here is the equivalent configuration in V3-style. Notice that the Validation Ope
...
+- [How to configure a new Checkpoint using test_yaml_config](../../guides/validation/checkpoints/how_to_configure_a_new_checkpoint_using_test_yaml_config.md)
```
Comment (austiezr) [inline docs/guides/miscellaneous/migration_guide.md]: `../validation/<path>`

File: docs/guides/miscellaneous/migration_guide.md [hunk 16]
```diff
@@ -508,8 +508,8 @@ Here is the equivalent configuration in V3-style. Notice that the Validation Ope
...
```
Comment (austiezr) [inline docs/guides/miscellaneous/migration_guide.md]: `../validation/<path>`

File: docs/guides/validation/checkpoints/how_to_create_a_new_checkpoint.md [hunk 17]
```diff
@@ -52,7 +52,7 @@ validations:
 """
 ````
 
-This is the minimum required to configure a Checkpoint that will run the Expectation Suite `my_suite` against the data asset `MyDataAsset`. See [How to configure a new Checkpoint using test_yaml_config](/docs/guides/validation/checkpoints/how_to_configure_a_new_checkpoint_using_test_yaml_config) for advanced configuration options.
+This is the minimum required to configure a Checkpoint that will run the Expectation Suite `my_suite` against the data asset `MyDataAsset`. See [How to configure a new Checkpoint using test_yaml_config](../../../guides/validation/checkpoints/how_to_configure_a_new_checkpoint_using_test_yaml_config.md) for advanced configuration options.
```
Comment (austiezr) [inline docs/guides/validation/checkpoints/how_to_create_a_new_checkpoint.md]: `../../guides/<path>`

File: docs/guides/validation/checkpoints/how_to_create_a_new_checkpoint.md [hunk 18]
```diff
@@ -97,9 +97,9 @@ Before running a Checkpoint, make sure that all classes and Expectation Suites r
...
 
-For more advanced configurations of Checkpoints, please see [How to configure a new Checkpoint using test_yaml_config](/docs/guides/validation/checkpoints/how_to_configure_a_new_checkpoint_using_test_yaml_config).
+For more advanced configurations of Checkpoints, please see [How to configure a new Checkpoint using test_yaml_config](../../../guides/validation/checkpoints/how_to_configure_a_new_checkpoint_using_test_yaml_config.md).
 
 ### Additional Resources
 
-* [How to configure a new Checkpoint using test_yaml_config](/docs/guides/validation/checkpoints/how_to_configure_a_new_checkpoint_using_test_yaml_config)
-* [How to add validations data or suites to a Checkpoint](/docs/guides/validation/checkpoints/how_to_add_validations_data_or_suites_to_a_checkpoint)
+* [How to configure a new Checkpoint using test_yaml_config](../../../guides/validation/checkpoints/how_to_configure_a_new_checkpoint_using_test_yaml_config.md)
+* [How to add validations data or suites to a Checkpoint](../../../guides/validation/checkpoints/how_to_add_validations_data_or_suites_to_a_checkpoint.md)
```
Comment (austiezr) [inline docs/guides/validation/checkpoints/how_to_create_a_new_checkpoint.md]: `../../guides/<path>`

File: docs/guides/validation/validation_actions/how_to_collect_openlineage_metadata_using_a_validation_action.md [hunk 19]
```diff
@@ -16,7 +16,7 @@ This guide will explain how to use a Validation Action to emit results to an Ope
 <Prerequisites>
 
  - Created at least one Expectation Suite.
- - Created at least one [Checkpoint](/docs/guides/validation/checkpoints/how_to_create_a_new_checkpoint) - you will need it in order to test that the OpenLineage Validation Operator is working.
+ - Created at least one [Checkpoint](../../../guides/validation/checkpoints/how_to_create_a_new_checkpoint.md) - you will need it in order to test that the OpenLineage Validation Operator is working.
```
Comment (austiezr) [inline docs/guides/validation/validation_actions/how_to_collect_openlineage_metadata_using_a_validation_action.md]: `../../validation/<path>`

File: docs/reference/data_discovery.md [hunk 20]
```diff
@@ -4,10 +4,10 @@ title: Data Discovery
 
 
 Profiling is a way of Rendering Validation Results to produce a summary of observed characteristics. When Validation
-Results are rendered as Profiling data, they create a new section in [Data Docs](/docs/reference/data_docs). By computing the
+Results are rendered as Profiling data, they create a new section in [Data Docs](../reference/data_docs.md). By computing the
 **observed** properties of data, Profiling helps to understand and reason about the data's **expected** properties.
 
-To produce a useful data overview, Great Expectations uses a [profiler](/docs/reference/profilers) to build a special Expectation
+To produce a useful data overview, Great Expectations uses a [profiler](../reference/profilers.md) to build a special Expectation
```
Comment (austiezr) [inline docs/reference/data_discovery.md]: `./data_docs.md`

`./profilers.md`

File: docs/reference/data_discovery.md [hunk 21]
```diff
@@ -28,4 +28,4 @@ Profiling is still a beta feature in Great Expectations. Over time, we plan to e
 Warning: `BasicDatasetProfiler` will evaluate the entire batch without limits or sampling, which may be very time
 consuming. As a rule of thumb, we recommend starting with small batches of data.
 
-See the docs on [Profilers](/docs/reference/profilers) for more information.
+See the docs on [Profilers](../reference/profilers.md) for more information.
```
Comment (austiezr) [inline docs/reference/data_discovery.md]: `./profilers.md`

File: docs/reference/validation.md [hunk 22]
```diff
@@ -6,7 +6,7 @@ title: Validation
 Once you've constructed and stored Expectations, you can use them to validate new data. Validation generates a report
 that details any specific deviations from expected values.
 
-We recommend using a [Data Context](/docs/reference/data_context) to manage Expectation Suites and coordinate validation across runs.
+We recommend using a [Data Context](../reference/data_context.md) to manage Expectation Suites and coordinate validation across runs.
```
Comment (austiezr) [inline docs/reference/validation.md]: `./data_context.md`

**Review by austiezr — CHANGES_REQUESTED** [review]
Awesome to see this come in; thank you so much @andyjessen !

I've noted out places where the updated links are broken; otherwise love to see this. 🚀

File: docs/guides/validation/checkpoints/how_to_create_a_new_checkpoint.md (hunk 18 above)
Comment (andyjessen) [reply docs/guides/validation/checkpoints/how_to_create_a_new_checkpoint.md]: I couldn't get suggested path to work.

File: docs/guides/validation/checkpoints/how_to_create_a_new_checkpoint.md (hunk 17 above)
Comment (andyjessen) [reply docs/guides/validation/checkpoints/how_to_create_a_new_checkpoint.md]: I couldn't get suggested path to work.

File: docs/guides/connecting_to_your_data/components/next_steps.md (hunk 7 above)
Comment (andyjessen) [reply docs/guides/connecting_to_your_data/components/next_steps.md:3]: I couldn't get suggested path to work.

File: docs/contributing/contributing_checklist.md (hunk 4 above)
Comment (andyjessen) [reply docs/contributing/contributing_checklist.md]: I made the fix in new commit.

File: docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md (hunk 5 above)
Comment (andyjessen) [reply docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md]: I made the fix in new commit.

File: docs/intro.md (hunk 1 above)
Comment (andyjessen) [reply docs/intro.md]: I made the fix in new commit.

File: docs/tutorials/getting_started/customize_your_deployment.md (hunk 2 above)
Comment (andyjessen) [reply docs/tutorials/getting_started/customize_your_deployment.md]: I fixed the link.

File: docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md (hunk 6 above)
Comment (andyjessen) [reply docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md]: I made the fix in new commit.

File: docs/guides/miscellaneous/how_to_write_a_how_to_guide.md (hunk 8 above)
Comment (andyjessen) [reply docs/guides/miscellaneous/how_to_write_a_how_to_guide.md]: I made the fix in new commit.

File: docs/reference/data_discovery.md (hunk 21 above)
Comment (andyjessen) [reply docs/reference/data_discovery.md]: I made the fix in new commit.

File: docs/reference/validation.md (hunk 22 above)
Comment (andyjessen) [reply docs/reference/validation.md]: I made the fix in new commit.

File: docs/reference/data_discovery.md (hunk 20 above)
Comment (andyjessen) [reply docs/reference/data_discovery.md]: I made the fix in new commit.

File: docs/guides/miscellaneous/migration_guide.md (hunk 9 above)
Comment (andyjessen) [reply docs/guides/miscellaneous/migration_guide.md]: I made the fix in another commit.

File: docs/guides/miscellaneous/migration_guide.md (hunk 10 above)
Comment (andyjessen) [reply docs/guides/miscellaneous/migration_guide.md]: I made the fix in another commit.

File: docs/guides/miscellaneous/migration_guide.md (hunk 11 above)
Comment (andyjessen) [reply docs/guides/miscellaneous/migration_guide.md]: I made the fix in another commit.

File: docs/guides/miscellaneous/migration_guide.md (hunk 12 above)
Comment (andyjessen) [reply docs/guides/miscellaneous/migration_guide.md]: I made the fix in another commit.

File: docs/guides/miscellaneous/migration_guide.md (hunk 13 above)
Comment (andyjessen) [reply docs/guides/miscellaneous/migration_guide.md]: I made the fix in another commit.

File: docs/guides/miscellaneous/migration_guide.md (hunk 14 above)
Comment (andyjessen) [reply docs/guides/miscellaneous/migration_guide.md]: I made the fix in another commit.

File: docs/guides/miscellaneous/migration_guide.md (hunk 15 above)
Comment (andyjessen) [reply docs/guides/miscellaneous/migration_guide.md]: I made the fix in another commit.

File: docs/guides/validation/validation_actions/how_to_collect_openlineage_metadata_using_a_validation_action.md (hunk 19 above)
Comment (andyjessen) [reply docs/guides/validation/validation_actions/how_to_collect_openlineage_metadata_using_a_validation_action.md]: I made the fix in another commit.

File: docs/guides/miscellaneous/migration_guide.md (hunk 16 above)
Comment (andyjessen) [reply docs/guides/miscellaneous/migration_guide.md]: I made the fix in another commit.

File: docs/tutorials/getting_started/customize_your_deployment.md (hunk 3 above)
Comment (andyjessen) [reply docs/tutorials/getting_started/customize_your_deployment.md]: I fixed in another commit.

File: docs/guides/validation/checkpoints/how_to_create_a_new_checkpoint.md (hunk 18 above)
Comment (austiezr) [reply docs/guides/validation/checkpoints/how_to_create_a_new_checkpoint.md]: My bad; looks like these should be `./<file_name>`

File: docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md [hunk 23]
```diff
@@ -324,8 +324,8 @@ We will now take the local GE configuration from [Part 1](#part-1-local-configur
 
 There are a number of ways that Great Expectations can be run in Cloud Composer or Airflow.
 
-1. [Running a Checkpoint in Airflow using a `bash operator`](/docs/deployment_patterns/how_to_use_great_expectations_with_airflow#option-1-running-a-checkpoint-with-a-bashoperator)
-2. [Running a Checkpoint in Airflow using a `python operator`](/docs/deployment_patterns/how_to_use_great_expectations_with_airflow#option-2-running-the-checkpoint-script-output-with-a-pythonoperator)
+1. [Running a Checkpoint in Airflow using a `bash operator`](./how_to_use_great_expectations_with_airflow#option-1-running-a-checkpoint-with-a-bashoperator.md)
+2. [Running a Checkpoint in Airflow using a `python operator`](./how_to_use_great_expectations_with_airflow#option-2-running-the-checkpoint-script-output-with-a-pythonoperator.md)
```
Comment (austiezr) [inline docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md:328]: `./<path>.md#<heading>`

Comment (austiezr) [inline docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md]: `./<path>.md#<heading>`

File: docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md [hunk 24]
```diff
@@ -451,14 +451,14 @@ You've successfully migrated your Great Expectations configuration to Cloud Comp
 
 There are many ways to iterate and improve this initial version, which used a `bash operator` for simplicity. For information on more sophisticated ways of triggering Checkpoints, building our DAGs, and dividing our Data Assets into Batches using DataConnectors, please refer to the following documentation: 
 
-- [How to run a Checkpoint in Airflow using a `python operator`](/docs/deployment_patterns/how_to_use_great_expectations_with_airflow#option-2-running-the-checkpoint-script-output-with-a-pythonoperator).
+- [How to run a Checkpoint in Airflow using a `python operator`](./how_to_use_great_expectations_with_airflow#option-2-running-the-checkpoint-script-output-with-a-pythonoperator.md).
```
Comment (austiezr) [inline docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md]: `./<path>.md#<heading>`

**Review by austiezr — CHANGES_REQUESTED** [review]
I believe these are the last pieces; thank you for your effort on this! 🙇

File: docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md (hunk 23 above)
Comment (andyjessen) [reply docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md]: I fixed link in new commit.

File: docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md (hunk 24 above)
Comment (andyjessen) [reply docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md]: I fixed link in new commit.

**Comment by andyjessen** [comment]
> I believe these are the last pieces; thank you for your effort on this! 🙇

Hi @austiezr  - I fixed links. I noticed another broken path in same file and fixed it as well.

File: docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md (hunk 24 above)
Comment (austiezr) [reply docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md]: Hey @andyjessen ! I'm not seeing the latest commit re: above.

Comment (andyjessen) [reply docs/deployment_patterns/how_to_use_great_expectations_with_google_cloud_platform_and_bigquery.md]: Thanks @austiezr for catching my mistake. (I made commit on separate branch that got pruned.) I added to current branch and pushed.

**Review by austiezr — APPROVED** [review]
Killer, thank you! 🙇

**Pull request merged**